src/Sharpliner/AzureDevOps/ConditionedExpressions/Arguments/InlineStringOrVariableOrParameter.cs

```diff
@@ -0,0 +1,8 @@
+﻿using OneOf;
+
+namespace Sharpliner.AzureDevOps.ConditionedExpressions.Arguments;
+
+[GenerateOneOf]
+public partial class InlineStringOrVariableOrParameter : OneOfBase<string, ParameterReference, VariableReference>
```

Comment:
Can we think of a better name for this? Maybe `InlineExpression` or similar?
The same for `IfString..`